antarest/study/business/utils.py

```diff
@@ -37,7 +38,11 @@ def execute_or_add_commands(
     commands: t.Sequence[ICommand],
     storage_service: StudyStorageService,
     listener: t.Optional[ICommandListener] = None,
+    jwt_user: t.Optional[JWTUser] = None,
 ) -> None:
+    # get current user if not in session, otherwise get session user
+    current_user = get_current_user() or jwt_user
```

Comment:
Here, if the task service correctly fills the current user in the task execution thread, we should not be able to rely only on `get_current_user()`